setups/init_submodules.sh

```
@@ -0,0 +1,13 @@
+#!/bin/bash
+# HOWTO: Run the script from the main directory
+# ./setups/init_submodules.sh
+echo "Init submodules..."
+git submodule update --init --recursive
+
+CADMesh_VERSION=2.0.3
+echo "CADMesh version: ${CADMesh_VERSION}"
+cd Submodules/CADMesh;git checkout tags/v${CADMesh_VERSION};cd ../..
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)